src/main/java/seedu/address/model/person/Name.java

```diff
@@ -49,19 +49,16 @@ public boolean equals(Object other) {
         if (other == this) {
             return true;
         }
-
-        // instanceof handles nulls
         if (!(other instanceof Name)) {
             return false;
         }
 
         Name otherName = (Name) other;
-        return fullName.equals(otherName.fullName);
+        return fullName.equalsIgnoreCase((otherName.fullName));
```

Comment:
I like this change! It would be annoying for our user to have a name that cannot be matched due to a typo.